namespaces/cloud-platform-test-1.k8s.integration.dsd.io/traefik/traefik-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: traefik-ingress-controller
+  namespace: kube-system
```

Comment:
The whole thing should live in the `ingress-controllers` namespace. This ensures that no further configuration is needed when it comes to `NetworkPolicies`.